lib/plugins/http-error-handler.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+function httpErrorHandler(request, reply) {
+  const response = request.response;
+  if (response.isBoom) {
+    let msg;
+    let statusCode = response.output.payload.statusCode;
+    let errorTitle = response.output.payload.error;
+    let errorMessage = response.output.payload.message;
+
+    switch(statusCode) {
+      case 404:
+        errorMessage = 'Sorry, that page is not available.';
```

Comment:
Could you explain your reasoning for overwritting the 404 error messages here? Why don't you use the default message?